tests/testthat/test-formatting.R

```diff
@@ -37,7 +37,7 @@ test_that("table_shell works", {
                   rrow("Hiya", "xx (xx.x%)", "xx.x - xx.x"),
                   rrow("bye", "xx.xx", "xx.xx"))
 
-    expect_identical(toString(tblsh),
+    expect_identical(toString(tblsh), 
```

Comment:
whitespace change

Reply:
yes, I usually run `styler:::style_selection()` or `styler:::style_active_pkg()` to avoid checking every whitespace manually, but it seems you do not have a general coding standard adopted, or at least I do not know which one. If I run the above commands it is using `tidyverse` standard and it is changing basically all the code. For example, my Rstudio-server does not recognize the double spaces as single tabs, so I have to double space every line manually which is quite annoying and prone to errors. I know you can set up different standards but this would not be in line with `tern` and all the other packages that use `styler`. I suggest adopting the above general standard, also to improve readability. We can discuss this later today. For the moment, of course, I will fix these two white spaces